### ISSUE TITLE
Simplify chunks (2) - wrap up qenv

### ISSUE THREAD

**Issue by kumamiao** [insightsengineering/NEST-roadmap#69]
A continuation of https://github.com/insightsengineering/NEST-roadmap/issues/45

@nikolas-burkoff provided a summary of what still need to be done [here](https://github.com/insightsengineering/teal/issues/731), but for simplicity and better refinement process, I will bring those items to this roadmap so we can see everything in one place:

### Items to do so `teal_refactor` can be merged to `main`:
- [x] https://github.com/insightsengineering/teal.code/issues/63
     -teal.code: https://github.com/insightsengineering/teal.code/pull/79
     -teal.modules.general: https://github.com/insightsengineering/teal.modules.general/pull/465
     -teal.modules.clinical: https://github.com/insightsengineering/teal.modules.clinical/pull/621
     -teal.goshawk: https://github.com/insightsengineering/teal.goshawk/pull/191
     -teal.osprey: https://github.com/insightsengineering/teal.osprey/pull/181
     -teal: https://github.com/insightsengineering/teal/pull/739
- [x] https://github.com/insightsengineering/teal.code/issues/57
- [x] https://github.com/insightsengineering/teal.modules.general/issues/476 https://github.com/insightsengineering/teal.modules.general/pull/484
- [x] open PRs from teal_refactor to main in teal.code, teal, tmhermes, tmhelios, tmg, tmc, t.g, t.o - see https://github.com/insightsengineering/teal/issues/731
- [x] https://github.com/insightsengineering/teal/issues/750
- [x] https://github.com/insightsengineering/teal.code/issues/86
- [x] https://github.com/insightsengineering/teal.modules.clinical/issues/668
- [x] https://github.com/insightsengineering/teal.modules.clinical/issues/670
- [x] app/module testing
- [x] make PRs green and approved

### Items to do after `teal_refactor` is merged:

- [x] https://github.com/insightsengineering/teal.widgets/issues/102
- [ ] ~~https://github.com/insightsengineering/teal.code/issues/84~~
- [x] https://github.com/insightsengineering/teal.code/issues/56
- [x] https://github.com/insightsengineering/teal.code/issues/4
- [x] https://github.com/insightsengineering/teal/issues/751
- [x] https://github.com/insightsengineering/teal/issues/752
- [x] https://github.com/insightsengineering/teal.slice/issues/84
    - [x] Decide on the teal.slice PR to turn on/off the filter panel so that functionality from data.table and variable browser is not lost https://github.com/insightsengineering/teal.slice/pull/91
- [x] https://github.com/insightsengineering/teal.code/issues/89

### Item still needs to be refined for next development:
- [ ] ~~https://github.com/insightsengineering/teal/issues/769~~
- [ ] ~~https://github.com/insightsengineering/teal/issues/937~~

**Comment by nikolas-burkoff** [comment]
Not all of the above need to be done before merging (and https://github.com/insightsengineering/teal.data/issues/102 absolutely shouldn't be)

**Comment by donyunardi** [comment]
> Not all of the above need to be done before merging (and [insightsengineering/teal.data#102](https://github.com/insightsengineering/teal.data/issues/102) absolutely shouldn't be)

Updated the comment. Thanks!

**Comment by donyunardi** [comment]
Closing.
Move rest of open issues to new card: insightsengineering/nestdevs-tasks#2
